### PR TITLE
Create Remark class for Order

### LINKED ISSUE #31: Add order is successful when there is no remark
Bug spotted:

`addorder 1 d/ 2020-01-01 r/ ` is successful when it should not be, since remark is empty

### PR DESCRIPTION
**Pull request by guanquann**
- To standardise the way we create attributes for classes.  
- Fix #31 as well

### CODE REVIEW

**Review by sun-ruiheng — COMMENTED** [review]
Looks good otherwise!

File: src/main/java/seedu/address/model/order/Status.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        requireNonNull(status);
+        checkArgument(isValidStatus(status), MESSAGE_CONSTRAINTS);
+        this.status = status;
+    }
+
+    /**
+     * Returns true if a given string is a valid status.
+     */
+    public static boolean isValidStatus(String test) {
+        return test.equals("Pending") || test.equals("Arrived") || test.equals("Late");
```
[the remark this answers is not in the document]
Comment (guanquann) [reply src/main/java/seedu/address/model/order/Status.java:34]: Hmm, I'm facing some issues with using .values() or any related methods that get all the enum. 
https://stackoverflow.com/questions/63253645/is-there-a-workaround-for-not-being-able-to-access-static-methods-in-an-enum-ins

EnumUtils is not installed by default as well. 

😢😢😢

[the remark this answers is not in the document]
Comment (guanquann) [reply src/main/java/seedu/address/model/order/Status.java]: Fixed in latest commit

**Review by maze508 — APPROVED** [review]
LGTM

**Pull request merged**